var/spack/repos/builtin/packages/openssh/package.py

```diff
@@ -51,6 +51,8 @@ class Openssh(AutotoolsPackage):
     depends_on('ncurses')
     depends_on('zlib')
     depends_on('py-twisted', type='test')
+    depends_on('krb5', when='+gssapi')
+    variant("gssapi", default=True)
```

Comment:
Can we put the `variant` directive before all of the `depends_on` directives? Also, we need at least a `description=` argument for the variant.

Reply:
I propose "Enable authentication via Kerberos through GSSAPI" for the description.

Reply:
Thanks, applied!